cookbooks/aws-parallelcluster-environment/files/cloudwatch/write_cloudwatch_agent_json.py

```diff
@@ -64,7 +64,7 @@ def add_instance_log_stream_prefixes(configs):
 
 
 def read_data(config_path):
-    """Read in log configuration data from config_path."""
+    """Read in and render jinja template of the log configuration data from config_path."""
```

Comment:
Why not add this in the `cookbooks/aws-parallelcluster-environment/files/cloudwatch/cloudwatch_agent_common_utils.py` if its being used in bot the other Python modules while keeping the comments to explain w.r.t these python modules